libs/blocks/aside/aside.css

```diff
@@ -7,9 +8,14 @@
   overflow: hidden;
 }
 
+.dark .aside {
+  color: white;
+}
+
 .aside .detail-M {
   text-transform: uppercase;
   font-weight: 700;
```

Comment:
Thinks we can omit these text styles. 
Any `detail-*` selector should already have these styles from the typography.css base styles.